src/debugger/Stepper.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+import type { StoppedData } from "./types.js";
+import { IsStatement } from "./../methods/is.js";
+import { BabelNode } from "babel-types";
+import invariant from "./../invariant.js";
+
+export class Stepper {
+  _prevStopData: StoppedData;
+  isComplete(ast: BabelNode): boolean {
+    invariant(false, "Abstract method, please override");
+  }
+}
+
+export class StepperInto extends Stepper {
+  constructor(filePath: string, line: number, column: number) {
+    super();
+    this._prevStopData = {
+      filePath: filePath,
+      line: line,
+      column: column,
+    };
+  }
+
+  isComplete(ast: BabelNode): boolean {
```

Comment:
This should be a common API among all steppers. Put this as an abstract method in base class which throws exception and override here.

Reply:
This is already the case. I will add // Override here.